tests/tests/test_legal_reason.py

```diff
@@ -0,0 +1,318 @@
+from django.contrib.contenttypes.models import ContentType
+from django.test import TestCase
+from faker import Faker
+
+from gdpr.models import LegalReason
+from tests.models import Account, Customer, Email, Payment
+from tests.purposes import (
+    ACCOUNT_AND_PAYMENT_SLUG, ACCOUNT_SLUG, EMAIL_SLUG, EVERYTHING_SLUG, EmailsPurpose, FIRST_AND_LAST_NAME_SLUG)
+from tests.tests.data import (
+    ACCOUNT__NUMBER, ACCOUNT__NUMBER2, ACCOUNT__OWNER, ACCOUNT__OWNER2, CUSTOMER__BIRTH_DATE, CUSTOMER__EMAIL,
+    CUSTOMER__EMAIL2, CUSTOMER__EMAIL3, CUSTOMER__FB_ID, CUSTOMER__FIRST_NAME, CUSTOMER__IP, CUSTOMER__KWARGS,
+    CUSTOMER__LAST_NAME, CUSTOMER__PERSONAL_ID, CUSTOMER__PHONE_NUMBER)
+from tests.tests.utils import AnonymizedDataMixin, NotImplementedMixin
+
+
+class TestLegalReason(AnonymizedDataMixin, NotImplementedMixin, TestCase):
+    def setUp(self):
+        self.fake = Faker()
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.customer: Customer = Customer(**CUSTOMER__KWARGS)
```

Comment:
nebylo by lepší používat factories?

Reply:
Na ten jeden model co se tam použije dvakrát si myslímže je to zbytečné.